src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -64,6 +64,62 @@ private static string Concat(string container, string name)
             "System.Linq.Expressions.Expression`1" :
             "System.Linq.Expressions.Expression1`1";
         private static readonly string s_libPrefix = ExecutionConditionUtil.IsDesktop ? "mscorlib" : "netstandard";
+        private static readonly string s_corePrefix = ExecutionConditionUtil.IsDesktop ? "System.Core" : "netstandard";
+
+        private void TestDiagnosticsInMain(string mainBody, string[]? usings = null, TargetFramework targetFramework = TargetFramework.Standard, params DiagnosticDescription[] expectedDiagnostics)
```

Comment:
I'm not sure what these helpers are useful for. Why not just use top-level statements?

Reply:
I'm also not sure (this was Adam's code). But it seems you are right, top-level statements can be used instead: a3478b99d082c537a7cca022fc389235f8f8bcf2